prov/verbs/src/fi_verbs.h

```diff
@@ -327,6 +373,9 @@ struct fi_ibv_cq {
 	fi_ibv_trywait_func	trywait;
 	ofi_atomic32_t		nevents;
 	struct util_buf_pool	*wce_pool;
+
+	/* The IB XRC SRQ context associated with this CQ */
+	struct fi_ibv_srq_ep	*xrc_srq_ep;
```

Comment:
nit: I think we can get to this from the ep's attached to the cq.  If needed, as each ep is added, a function can check that the ep's srq matches with the ep srq at the head of the cq's list.

Reply:
sure

Reply:
Thanks. I have updated the code so that the XRC SRX context keeps the CQ used in creating it's XRC SRQ. The CQ now maintains a list of the SRX contexts where the underlying XRC SRQ was created using that CQ with commit78bc05e266537ee59563cdf8fdaff65d0ff8ae3e allowing it support being used with multiple SRX contexts.